settings.cmake

```diff
@@ -20,21 +20,17 @@ set(POLLY_DIR ${project_dir}/tools/polly CACHE INTERNAL "")
 
 include(application_settings)
 
-# Deal with the top level target-triplet variables.
-if(NOT TUT_BOARD)
-    message(
-        FATAL_ERROR
-            "Please select a board to compile for. Choose either pc or zynq7000\n\t`-DTUT_BOARD=<PREFERENCE>`"
-    )
-endif()
+set(TUT_BOARD "pc" CACHE STRING "Target machine to build the project for")
```

Comment:
I think it is still useful to tell the user about the default choice that are made and how these can be changed:
```
if(NOT TUT_BOARD)
    message("No board selected (-DTUT_BOARD=[pc|zynq7000]), defaulting to 'pc'")
    set(TUT_BOARD "pc" CACHE STRING "Target machine to build the project for")
endif()
```

Reply:
done